cmd/openshift-install/main.go

```diff
@@ -18,6 +22,16 @@ var (
 )
 
 func main() {
+	cname := strings.TrimSuffix(filepath.Base(os.Args[0]), ".exec")
```

Comment:
We may need a guard for `len(os.Args) > 0`, see [here][1].

[1]: https://github.com/openshift/installer/pull/919#discussion_r241933866

Reply:
> ... an example terraform binary doesn't have that guard...

Just because Terraform skips a check for a POSIX corner case doesn't mean we should ;).  It shouldn't be too bad:

```go
cname := "openshift-install"
if len(os.Args) > 0 {
  cname = filepath.Base(os.Args[0])
  extension := filepath.Ext(cname)
  cname = cname[0:len(cname)-len(extension)]
}
```

And that's with generic extention trimming.  If you wanted to stick with `.exe`, you could use:

```go
cname := "openshift-install"
if len(os.Args) > 0 {
  cname = strings.TrimSuffix(filepath.Base(os.Args[0]), ".exe")
}
```